### PR TITLE
Feature/python container

### PR DESCRIPTION
**Pull request by jiata**
python container

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,7 +114,7 @@ Most users will want to work interactively with their Spark clusters. With the `
 ```bash
 aztk spark cluster ssh --id <my_cluster_id>
 ```
-By default, we port forward the Spark Web UI to *localhost:8080*, Spark Jobs UI to *localhost:4040*, and Jupyter to *localhost:8888*.
+By default, we port forward the Spark Web UI to *localhost:8080* and Spark Jobs UI to *localhost:4040*.
```
Comment (paselem) [inline README.md]: change...

File: docker-image/README.md [hunk 2]
```diff
@@ -1,112 +1,109 @@
...
 
-### Supported Base Images
-We support several base images:
-- [Docker Hub] jiata/aztk:<image-version>-spark2.2.0-python3.5.4
-- [Docker Hub] jiata/aztk:<image-version>-spark2.1.0-python3.5.4
-- [Docker Hub] jiata/aztk:<image-version>-spark1.6.3-python3.5.4
-- [Docker Hub] jiata/aztk:<image-version>-spark2.1.0-python2.7.13
-- [Docker Hub] jiata/aztk:<image-version>-spark1.6.3-python2.7.13
+## Supported Images
+By default, this toolkit will use the Vanilla Spark image, __aztk-base__. This image contains the bare mininum to get Spark up and running in Standalone mode.
```
Comment (paselem) [inline docker-image/README.md]: Vanilla -> Base

Comment (paselem) [reply docker-image/README.md]: Standalone -> standalone

File: docker-image/README.md [hunk 3]
```diff
@@ -1,112 +1,109 @@
...
 
-NOTE: Replace **<image-version>** with the version of the image you wish to use. For example: **jiata/aztk:0.1.0-spark2.2.0-python3.5.4**
+On top of that, we also provide two flavors of Spark images, one geared towards the Python user (PySpark), and the other geared towards the R user (SparklyR or SparkR).
```
Comment (paselem) [inline docker-image/README.md]: Keep it to Pyspark for now?

@brnleehng  Also, do we have good support for SparkR really?

File: docker-image/README.md [hunk 4]
```diff
@@ -1,112 +1,109 @@
...
 
-### Gallery of 3rd Party Images
-Since this toolkit uses Docker containers to run Spark, users can bring their own images. Here's a list of 3rd party images:
-- *coming soon*
+Docker Image | Image Type | User Language(s) | What's Included? 
+:-- | :-- | :-- | :-- 
+[aztk-base](https://hub.docker.com/r/jiata/aztk-base/) | Base | Java, Scala |  `Spark`
+[aztk-python](https://hub.docker.com/r/jiata/aztk-python/) | Pyspark | Python | `Anaconda`</br>`Jupyter Notebooks` </br> `PySpark`
+[aztk-r](https://hub.docker.com/r/jiata/aztk-r/) | SparklyR | R | `CRAN`</br>`RStudio Server`</br>`SparklyR and SparkR`
```
Comment (paselem) [inline docker-image/README.md]: remove sparklyR for now

File: README.md (hunk 1 above)
Comment (jiata) [reply README.md]: what are we changing this to again?

Comment (paselem) [reply README.md]: history server to localhost:18080 (if you have enabled the history server)

File: docker-image/README.md [hunk 5]
```diff
@@ -1,112 +1,109 @@
...
+## Hosting your Docker Image
+By default, this toolkit assumes that your Docker images are publicly hosted on Docker Hub. However, we also support hosting your images privately.
+
+See [here](https://github.com/Azure/aztk/blob/master/docs/12-docker-image.md#using-a-custom-docker-image-that-is-privately-hosted) to learn more about using privately hosted Docker Images.
+
+## Learn More 
+The Dockerfiles in this directory are used to build the Docker images used by this toolkit. Please reference the individual directories for more information on each Dockerfile:
+- [Base](./base)
+- [Python](./python)
+- TODO: R
```
Comment (paselem) [inline docker-image/README.md]: TODO -> [coming soon]?

File: README.md (hunk 1 above)
Comment (jiata) [reply README.md]: right

File: docker-image/README.md (hunk 5 above)
Comment (jiata) [reply docker-image/README.md]: right

File: README.md (hunk 1 above)
Comment (jiata) [reply README.md]: should be in your PR... but i'll add it anyway

**Pull request merged**